src/index.html

```diff
@@ -1,13 +1,45 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html class= "page" lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="
+            width=device-width, 
+            user-scalable=no, 
+            initial-scale=1.0, 
+            maximum-scale=1.0, 
+            minimum-scale=1.0
+          ">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">    
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page__body">
+    <div class="card" data-qa="card">
+      <img src="./images/imac.jpeg" alt="iMac" class="card__img img">
+      <div class="card__description">
+        <div class="card__product">
+          <h2 class="card__header header">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__code code">Product code: 195434</p>
+        </div>
+        <div class="card__reviews reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>            
+            <div class="stars__star"></div>           
+            <div class="stars__star"></div>           
+            <div class="stars__star"></div>
+          </div>
+          <p class="reviews__number">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__text text">Price:</p>
+          <p class="card__price-number price-number">$2,199</p>
```

Comment:
Why do you need this `price-number` class here?
```suggestion
          <p class="card__price-number">$2,199</p>
```